### PR TITLE
fix: snags and bugs revealed by docs writing

### PR DESCRIPTION
**Pull request by robinbryce**
* There was a corner case in replicate-logs where --ancestors was equal or lestthe number of massifs. this is now fixed and has tests. Bug #9856
* The watch command had a superflous mode option. this has been removed. #9854
* The watch command output was reporting tenant uuids rather than identities, it now reports identities. Bug #9855
* The --tenant option was used inconistently by some sub commands, this is now always a global option. Individual commands are free to support a list of comma seperated tenants. If a list is provided to a command that only deals with one, the command will take the first. #9857
* All commands which read seals failed to use the production url by default. #9858

### CODE REVIEW

File: cfgtenants.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package veracity
+
+import (
+	"fmt"
+	"strings"
+)
+
+// NormTenantIdentity ensures a string is prefixed with 'tenant/'
+// Note the expected input is a uuid string or a tenant/uuid string
+func NormTenantIdentity(tenant string) string {
```
Comment (honourfish) [inline cfgtenants.go:10]: minor: I am not sure i like the word Norm, i understand we want to be robust incase we decide we hate the tenant/uuid format, but i think thats pretty much baked in, maybe call this something closer to `HasTenantPrefix`.

really minor, it is readable as it is.

File: replicatelogs.go [hunk 2]
```diff
@@ -271,91 +277,87 @@ func (v *VerifiedReplica) ReplicateVerifiedUpdates(
...
 	if !isNilOrNotFound(err) {
 		return err
 	}
 
-	var local *massifs.VerifiedContext
-
-	// We always verify up to the requested massif.
-	// ancestorCount is used to ensure there is a minimum number of verified massifs replicated locally.
-	// Our verification always reads the remote massifs starting from requested - ancestorCount.
+	// We always verify up to the requested massif, but we do not re-verify
```
Comment (honourfish) [inline replicatelogs.go:288]: great commentary! :)

**Review by honourfish — APPROVED** [review]

File: cfgtenants.go (hunk 1 above)
Comment (henry739) [reply cfgtenants.go:10]: Yeah, or Ensure, since its doing the transformation if its missing

File: tests/replicatelogs/replicatelogs_azurite_test.go [hunk 3]
```diff
@@ -76,9 +76,99 @@ func (s *ReplicateLogsCmdSuite) TestReplicatingMassifLogsForOneTenant() {
...
+				for i := uint32(0); i < massifCount; i++ {
+					expectMassifFile := filepath.Join(replicaDir, massifs.ReplicaRelativeMassifPath(tenantId0, i))
+					s.FileExistsf(expectMassifFile, "the replicated massif should exist")
+					expectSealFile := filepath.Join(replicaDir, massifs.ReplicaRelativeSealPath(tenantId0, i))
+					s.FileExistsf(expectSealFile, "the replicated seal should exist")
+				}
+				return
+			}
+
+			end := max(2, massifCount) - 2 - tt.ancestors
```
Comment (henry739) [inline tests/replicatelogs/replicatelogs_azurite_test.go:152]: A brief comment here explaining why we're doing `max(2, massifCount) - 2` would be great

File: cfgtenants.go (hunk 1 above)
Comment (henry739) [reply cfgtenants.go:10]: Or just Normalize tbh

Comment (robinbryce) [reply cfgtenants.go:10]: I have a follow up PR to do to add some tests for the release. I'll pick this up there
